embedded/drivers/include/can.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __CAN_H__
+#define __CAN_H__
+
+#include <linux/can.h>
+#include <stdint.h>
+
+class Can {
+ private:
+  int sock;
```

Comment:
We need to create a mutex on the can bus, since there _will_ be multiple devices accessing it. I will take care of this